src/components/RegionCard/styles.ts

```diff
@@ -4,20 +4,19 @@ import {card} from '../../styles/card'
 
 export const regionCard = classes(
```

Comment:
Could you add: "cursor: pointer", so it shows as clickable on desktop?

Reply:
Over the export?

Reply:
Just within the regionCard styling somewhere :)